app/view/show.py

```diff
@@ -19,6 +19,9 @@
 
 
 def get_bug_data(cves, pkgs, group):
+
+    MAXLENGTH_SUMMARY = 200
```

Comment:
`# the form's summary of bugs.archlinux.org has a maximum field length of 200` should be fine